meerkat_abacus/consumer/tests/__init__.py

```diff
@@ -7,6 +7,10 @@
 from unittest import mock
 import random
 import unittest
+import sys
+sys.modules[
+    'meerkat_abacus.pipeline_worker.processing_tasks'
+] = mock.MagicMock()
```

Comment:
That looks very hacky. Is that a "permanent" solution?

Reply:
The idea here is to avoid having to import the processing_tasks module as we are just mocking it anyway. If there is a nicer way of doing that it would be great.